libpod/image/pull.go

```diff
@@ -149,6 +149,13 @@ func (ir *Runtime) pullGoalFromImageReference(ctx context.Context, srcRef types.
 		// Need to load in all the repo tags from the manifest
 		res := []pullRefPair{}
 		for _, dst := range manifest[0].RepoTags {
+			//check if image exists and gives a warning of untagging
+			localImage, err := ir.NewFromLocal(dst)
+			imageID := strings.TrimSuffix(manifest[0].Config, ".json")
+			if err == nil && imageID != localImage.ID() {
+				logrus.Errorf("the image %s already exists, renaming the old one with ID %s to empty string", dst, localImage.ID())
```

Comment:
"renaming" implies the software is in the process of changing the images name which doesn't seem to be happening?
Perhaps:
"the image %s already exists, rename the image with ID %s to another value and retry"

There's probably a better way to say the above, but that's all I got after one cup of tea.

Reply:
Docker only gives this warning and continues to untag it, so I only added the warning